README.md

```diff
@@ -1,93 +1,104 @@
-thus can achieve high parallelism for a large SAS file.
+A library for reading SAS data (.sas7bdat) with [Spark](http://spark.apache.org/). 
 
-This library is inspired by [spark-csv](https://github.com/databricks/spark-csv) and
-currently uses [parso](https://github.com/epam/parso/) for parsing as it is the only public available parser
-that handles both forms of SAS compression (CHAR and BINARY). 
-
-[![Build Status](https://travis-ci.org/saurfang/spark-sas7bdat.svg?branch=master)](https://travis-ci.org/saurfang/spark-sas7bdat) [![Join the chat at https://gitter.im/saurfang/spark-sas7bdat](https://badges.gitter.im/saurfang/spark-sas7bdat.svg)](https://gitter.im/saurfang/spark-sas7bdat?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-## Requirements
 
-This library requires Spark 1.4+
-
-## How To Use
+[![Build Status](https://travis-ci.org/saurfang/spark-sas7bdat.svg?branch=master)](https://travis-ci.org/saurfang/spark-sas7bdat) [![Join the chat at https://gitter.im/saurfang/spark-sas7bdat](https://badges.gitter.im/saurfang/spark-sas7bdat.svg)](https://gitter.im/saurfang/spark-sas7bdat?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-This package is published using [sbt-spark-package](https://github.com/databricks/sbt-spark-package) and
-linking information can be found at http://spark-packages.org/package/saurfang/spark-sas7bdat
+## Requirements:
+* [Spark 1.4+ or 2.0+](https://spark.apache.org/downloads.html)
+* [Parso 2.0.10](https://mvnrepository.com/artifact/com.epam/parso/2.0.10)
 
-## Features
+## Download:
 
-This package allows reading SAS files in local or distributed filesystem as
-[Spark DataFrames](https://spark.apache.org/docs/latest/sql-programming-guide.html).
+The latest jar can be downloaded from [spark-packages](http://spark-packages.org/package/saurfang/spark-sas7bdat).
 
-Schema is automatically inferred from meta information embedded in the SAS file.
+## Features:
 
-Thanks to the splittable `SasInputFormat`, we are able to convert a 200GB (1.5Bn rows) .sas7bdat file
-to .csv files using 2000 executors in under 2 minutes.
+* This package allows reading SAS files from local and distributed filesystems, into Spark DataFrames.
+* Schema is automatically inferred from metadata embedded in the SAS file. _(Behaviour can be customised, see parameters below)_
+* The SAS format is splittable when not file-system compressed, thus we are able to convert a 200GB (1.5Bn rows) .sas7bdat file to .csv files using 2000 executors in under 2 minutes.
+* This library uses [parso](https://github.com/epam/parso/) for parsing as it is the only public available parser
+that handles both forms of SAS compression (CHAR and BINARY). 
 
-### SQL API
-SAS data can be queried in pure SQL by registering the data as a (temporary) table.
-
-```sql
-CREATE TEMPORARY TABLE cars
-USING com.github.saurfang.sas.spark
-OPTIONS (path "cars.sas7bdat")
-```
+__NOTE:__ we currently only support reading
```

Comment:
did you mean to say something here?

Reply:
I mean to say we don't current sport writing sas7bdat, but I can make that more clear.